uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/calendar-feeds/link-modal/calendar-feeds-link-modal.html

```diff
@@ -0,0 +1,18 @@
+<div class="calendar-feeds-link-modal-component">
+  <div class="modal-header">
+    <button type="button" class="close" ng-click="linkModal.dismiss()">
+      <span aria-hidden="true">&times;</span><span class="sr-only">Close</span>
+    </button>
+    <h4 class="modal-title">Calendar feed link:</h4>
+  </div>
+  <div class="modal-body">
+    <input-with-copy-button ng-model="linkModal.url"></input-with-copy-button>
+    <br />
```

Comment:
Can we use divs and not brs unless it is text?

Reply:
If we use divs we won't have spacing between the input/button and the link.

Reply:
I think we should find a bit more elegant solution to add spacing. Maybe the `form-group` class?